lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -17,7 +17,8 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const element = this.dom.find('.lh-metric', tmpl);
     element.id = audit.result.name;
     // FIXME(paulirish): currently this sets a 'lh-metric--fail' class on error'd audits
```

Comment:
can you also address this?

i'd prefer do not set a --fail class on these audits.

Reply:
haha sure